addons/web/static/src/views/list/list_confirmation_dialog.xml

```diff
@@ -7,10 +7,11 @@
                 <p>
                     <t t-if="props.isDomainSelected">This update will only consider the records of the current page.<br/><br/></t>
                     <t t-if="props.nbRecords != props.nbValidRecords">
-                        Among the <t t-esc="props.nbRecords"/> selected records,
-                        <t t-esc="props.nbValidRecords"/> are valid for this update.<br/>
+                        <t t-esc="validRecordsText"/>
+                        <br/>
                     </t>
-                    Are you sure you want to perform the following update on those <t t-esc="props.nbValidRecords"/> records?
+                    <t t-esc="updateConfirmationText"/><br/>
+                    <span class="text-muted"><strong><i>Enjoy how well designed Odoo is :)</i></strong></span>
```

Comment:
Uhh, this line looks _extremely_ awkward in context

![image](https://github.com/user-attachments/assets/c05abe36-24cb-4bf8-863c-7f23075af702)

Is the sentence really needed?

Reply:
> Uhh, this line looks _extremely_ awkward in context

Totally agree. I don't really understand the purpose in this context 🤔

Reply:
@jija2  If you have any suggestions, please do share.

Reply:
> @ppr-odoo can you remove that line?
> 
> cc @aju-odoo

:heavy_check_mark: